flexget/plugins/input/smart_imdb.py

```diff
@@ -0,0 +1,478 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+import re
+from jsonschema.compat import str_types
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+from imdb import IMDb
```

Comment:
I updated based on this feedback, but now i'm thinking i should have done the whole thing different, and split this into two plugins. one for the input and one for the filter... 
what do you think?